From: Simon McVittie <smcv@collabora.com>
Date: Mon, 18 Jan 2021 11:22:34 +0000
Subject: spirv.pc: Add dependencies on SPIRV-Tools and glslang

Otherwise, a simple program like this will fail to link:

    #include <glslang/SPIRV/GlslangToSpv.h>
    int main (void)
    {
      std::string s;
      glslang::GetSpirvVersion(s);
      return 0;
    }

when compiled with the obvious parameters from pkg-config:

    g++ -ospirv spirv.cpp $(pkg-config --cflags --libs spirv)

Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=951988
Signed-off-by: Simon McVittie <smcv@collabora.com>
---
 SPIRV/spirv.pc.cmake.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/SPIRV/spirv.pc.cmake.in
+++ b/SPIRV/spirv.pc.cmake.in
@@ -5,7 +5,7 @@ includedir=${prefix}/@CMAKE_INSTALL_INCL
 
 Name: @SPIRV_NAME@
 Description: SPIR-V is a binary intermediate language for representing graphical-shader stages and compute kernels for multiple Khronos APIs, including OpenCL, OpenGL, and Vulkan
-Requires:
+Requires: SPIRV-Tools, glslang
 Version: @SPIRV_VERSION@
 Libs: -L${libdir} -lSPIRV
 Cflags: -I${includedir}
